tests/test_ixps_import.py

```diff
@@ -28,16 +28,18 @@ def test_imports_a_new_ixp():
 def test_updates_an_existing_ixp():
     new_data = PeeringIXFactory()
     manrs_participants = [new_data["id"]]
-    IXPFactory(peeringdb_id=new_data["id"], last_active=datetime(year=2024, month=4, day=1).replace(tzinfo=timezone.utc), manrs_participant=False)
+    anchor_hosts = [new_data["id"]]
+    IXPFactory(peeringdb_id=new_data["id"], last_active=datetime(year=2024, month=4, day=1).replace(tzinfo=timezone.utc), manrs_participant=False, anchor_host=False)
 
-    importers.process_ixp_data(datetime.now(timezone.utc), MockLookup(manrs_participants=manrs_participants))([new_data])
+    importers.process_ixp_data(datetime.now(timezone.utc), MockLookup(manrs_participants=manrs_participants, anchor_hosts=anchor_hosts))([new_data])
 
     ixps = IXP.objects.all()
     assert len(ixps) == 1
     ixp = IXP.objects.first()
     assert ixp.last_active.date() == datetime.now(timezone.utc).date()
     assert ixp.name == new_data["name"]
     assert ixp.manrs_participant
+    assert ixp.anchor_host
```

Comment:
suggestion: I think we're testing the same thing twice here. We probably don't need both. I don't really mind which we keep (though perhaps standalone tests for this and MANRS participants are slightly cleaner).